test/integration/kubernetes_deploy_test.rb

```diff
@@ -707,10 +664,10 @@ def test_failed_deploy_to_nonexistent_namespace
     @namespace = original_ns
   end
 
-  def test_failure_logs_from_unmanaged_pod_appear_in_summary_section
+  def test_unmanaged_pod_failure_halts_deploy_and_displays_logs_correctly
```

Comment:
Modified this test to also cover the basic fact that it halts the deploy, which was previously covered by `test_bad_container_image_on_unmanaged_pod_halts_and_fails_deploy`.